packages/quantic/force-app/main/default/lwc/quanticStatefulButton/quanticStatefulButton.css

```diff
@@ -23,14 +23,14 @@
 }
 
 .stateful-button--selected {
-  --slds-c-button-outline-brand-color-border-hover: var(--main-color);
-  --slds-c-button-outline-brand-color-border: var(--main-color);
+  border-color: var(--main-color);
   --slds-c-button-shadow-focus: 0 0 3px var(--main-color);
 }
 
 .stateful-button--selected,
 .stateful-button--selected:hover,
 .stateful-button--selected:focus {
+  border-color: var(--main-color);
```

Comment:
please make sure to check the behaviour of the component in different states, on hover and when selected and compare the previous and the current behaviour.

Reply:
I did! it works fine. For the coming PRs I made sure I posted videos on styling influenced by interactions/behaviours of the user :P